elastic/integrations
# Fix CPU gauge to just use normalized CPU percentages

## What does this PR do?

This alters the CPU Usage Gauge to just use normalized values for visualizations, and removes the painless script, which was (incorrectly) dividing the normalized value by the CPU count.

I'd actually like some feedback on this, since I can't figure out why the visualization hasn't *always* been like this. We've been doing some math cleverness to take the "regular" CPU value and divide it by the CPU count, when this is basically what the `norm.cpu` values do to begin with. I've tested this change and it works fine, but I'm a tad paranoid and I wonder if there's a reason why we haven't been doing this before.

## Checklist


## How to test this PR locally

Pull down, build. Find the CPU Usage gauge In the System Integration Host Details dashboard. Make sure the gauge is populated and the value is correct.
